test/mocks/adapter/service/chargepoint/energy_aware_controller_helper.go

```diff
@@ -0,0 +1,25 @@
+package mockedchargepoint
+
+func MockEnergyAwareController() *EnergyAwareController {
+	return &EnergyAwareController{}
+}
+
```

Comment:
Prolly you should copy all helpers from controller here as well.

Reply:
Done.